lmnet/lmnet/datasets/div2k.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import functools
+import os
+from glob import glob
+
+from lmnet.datasets.base import Base
+from lmnet.utils.image import load_image
+
+
+class Div2k(Base):
+    classes = []
+    num_classes = 0
```

Comment:
I guess setting `num_classes = 0` is for base class restriction.
It's not clear. Please add comment.